vetur/attributes.json

```
@@ -13,6 +13,10 @@
     "type": "boolean",
     "description": "Setting replace prop will call `router.replace()` instead of `router.push()` when clicked, so the navigation will replace the current history entry.\n\n[API Reference](https://router.vuejs.org/api/#replace)"
   },
+  "custom": {
+    "type": "boolean",
+    "description": "Whether `RouterLink` should not wrap its content in an `a` tag."
```

I think using the kebab case annotation plus wrapping with `<>` will make it more familiar:
```suggestion
    "description": "Whether `<router-link>` should not wrap its content in an `<a>` tag."
```